frontend/src/pages/News/ShareYourStory.tsx

```diff
@@ -1,138 +1,93 @@
+          className='font-sansTitle text-bigHeader font-bold leading-lhNormal text-altGreen'
+        >
+          Share Your Story
+        </h1>
+        <section className='mx-4 flex flex-col items-center'>
+          <p className='max-w-md px-6 text-left'>
+            We believe that everyone's voice matters when it comes to health
+            equity and ending the HIV epidemic. We invite community members,
+            advocates, and individuals directly affected by HIV to share their
+            personal stories, insights, and experiences on our ‘News and
+            Stories’ page. By contributing, you can help raise awareness, foster
+            understanding, and inspire positive change in underserved
+            communities. To ensure the quality and credibility of the content,
+            we kindly ask you to follow the guidelines outlined below.
+          </p>
+          <HetButtonSecondary
+            buttonClassName='md:px-40 px-32 py-4 mb-10'
+            href='mailto:info@healthequitytracker.org'
+          >
+            Share your story
+          </HetButtonSecondary>
 
-          <li className='p-3'>
-            <b>Anonymity and Privacy:</b> If you prefer to remain anonymous or
-            use a pseudonym, please let us know in your submission email. We
-            respect your privacy and will handle your personal information with
-            utmost confidentiality.
-          </li>
+          <div className='mt-5 flex w-full items-center'>
+            <div className='flex-1 border-0 border-t border-solid border-altGrey'></div>
+            <h3 className='m-0 pe-4 ps-4 text-altGreen text-title md:text-smallestHeader font-bold md:font-medium leading-lhSomeMoreSpace'>
+              Submission guidelines
+            </h3>
+            <div className='flex-1 border-0 border-t border-solid border-altGrey'></div>
+          </div>
 
-          <li className='p-3'>
-            <b>Publication and Promotion:</b> While we cannot guarantee that all
-            submissions will be published, we appreciate your contribution and
-            will notify you if your story is selected. Published stories will be
-            promoted on our website and various social media platforms,
-            amplifying their reach and impact.
-          </li>
-        </ul>
+          <ul className='mt-4 mb-8 grid grid-cols-1 md:grid-cols-2 list-none ml-0 gap-2 max-w-md px-6 '>
+            {submissionGuidelines.map((submissionGuideline, index) => {
+              const isMobileShadow = !isMdAndUp && index % 2 === 0
+              const isDesktopShadow =
+                isMdAndUp &&
+                ((Math.floor(index / 2) % 2 === 0 && index % 2 === 0) ||
+                  (Math.floor(index / 2) % 2 !== 0 && index % 2 !== 0))
 
-        <div className='m-10 flex w-full items-center'>
-          <div className='flex-1 border-0 border-t border-solid border-altGrey'></div>
-        </div>
+              return (
+                <div
```

Comment:
this should be the `<li>` here, the direct child of the `<ul>` and then inside the CardLeftNumber component that can be a div or whatever. The a11y tester should have flagged that